src/components/newEvent/Overview.tsx

```diff
@@ -0,0 +1,247 @@
+            />
+            {error && <span className="error-message">{error.message}</span>}
+        </div>
+)};
+
+const UserSchema: ZodType<FormData> = z
+ .object({
+    eventName: z
+        .string()
+        .max(10),
+    startsAt: z
+        .string()
+        .date(),
+    startsAtTime: z
+        .string()
+        .time(),
+    endsAt: z
+        .string(),
+    endsAtTime: z
+        .string()
+        .time(),
+    type: z
+        .string(),
+    imageUrl: z.string(),
+});
+
+const TypeSelect = ({ onChange }: {onChange:any}) => {
+    const [selectedType, setSelectedType] = useState('online');
+  
+    const options = [
+      { value: 'online', label: 'Online' },
+      { value: 'inPerson', label: 'In person' },
+    ] as {
+        value: 'online' | 'inPerson',
+        label: 'Online' | 'In Person'
+    }[];
+
+    const ONLINE_TEXT = "Select this option if your event will be held online. Provide the access link or instructions at event description in the next step."
+    const IN_PERSON_TEXT = "Select this option if your event will take place in person. Include location details at event description in the next step."
+  
+    const handleSelect = (value:'online' | 'inPerson') => {
+      setSelectedType(value);
+      onChange(value);
+    };
+  
+    return (
+      <div className="mb-16 space-y-2">
+        <strong className="block">Type</strong>
+        <div className="grid grid-cols-2 gap-4">
+          {options.map((option) => (
+            <button
+              key={option.value}
+              type="button"
+              onClick={() => handleSelect(option.value)}
+              className={`
+                flex flex-col items-start p-4 rounded-lg border-2 transition-all duration-200 ease-in-out
+                ${
+                  selectedType === option.value
+                    ? 'border-blue-500 bg-blue-50 text-blue-700'
+                    : 'border-gray-300 hover:border-gray-400'
+                }
+              `}
+            >
+              <div className="font-semibold text-lg">{option.label}</div>
+              <div className="text-left text-sm text-gray-500">
+                {option.value === 'online' ? ONLINE_TEXT : IN_PERSON_TEXT}
+              </div>
+            </button>
+          ))}
+        </div>
+      </div>
+    );
+  };
+
+export function Overview() {
+
+    const {isContinueEnabled, setIsContinueEnabled} = useContext(newEventContext)
+
+    const {
+        register,
+        handleSubmit,
+        formState: { errors },
+        setError,
+        watch,
+      } = useForm<FormData>({
+        resolver: zodResolver(UserSchema), // Apply the zodResolver
+      });
+
+    function submitForm() {
+        console.log('Sucesso!')
+    }
+
+    const handleTypeChange = (value:React.FormEvent<HTMLInputElement>) => {
+        console.log('Selected type:', value);
+        // Update your form state here
+      };
+
+    const eventName = watch("eventName")
```

Comment:
You could just use `const isValid = await form.trigger();` 
```js
 const handleSubmit = async () => {

    const isValid = await form.trigger();
    if (!isValid) return;

  ....
  setIsContinueEnabled(true)
  };
```
you would need to set if the fields are required or not https://react-hook-form.com/docs/useform/register

Reply:
The button that calls the form is outside the form, and the form is not triggered. Kind of a wip situation, I will organize this better when joining the UI with the attestation creation